### PR TITLE
[Fix] CON-122 - advanced search modal match mockup

### PR DESCRIPTION
**Pull request by 404passionFruit**
**JIRA issue**
[CON-122 (comment after testing)](https://oat-sa.atlassian.net/browse/CON-122?focusedCommentId=114171)

**Related PR**
https://github.com/oat-sa/tao-core/pull/2772

**Fixies**
- [x] No space between “No matches found” and left border
Screenshot 2021-01-12 003039

- [x] The dropdown menu behave as input field

https://user-images.githubusercontent.com/11773029/104241112-33f8a100-546e-11eb-82b4-77580a52b7a1.mp4

- [x] The left panel with modal vies is shorter than the list of 20 results and has white blank space underneath

https://user-images.githubusercontent.com/11773029/104240499-35759980-546d-11eb-8582-b557c8f7b990.mp4

### CODE REVIEW

**Review by adriloma — APPROVED** [review]
Code review, looks good!

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] bumping versions are applied (if applicable)

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
lgtm , but i would like to see a demo ,  also please try to create a demo environment for QA testing

**Comment by 404passionFruit** [comment]
Added screenshot and videos to description

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: src/searchModal.js [hunk 1]
```diff
@@ -258,15 +258,50 @@ export default function searchModalFactory(config) {
...
 
         /**
-         * clicking on class filter input will toggle resource selector,
-         * will preventDefault to avoid focus on input field,
-         * and will stopPropagation to prevent be closed
+         * Pressing space, enter, esc, backspace 
+         * on class filter input will toggle resource selector
+         */
+        $classFilterInput.on('keyup', e => {
+            const spacePressed = e.code === "Space" || e.which === 32;
```
Comment (bziondik) [inline src/searchModal.js:265]: Please, check in tao-core-ui or tao-core-sdk some util/helper for key codes. If we have such helper, please, use it.

Comment (404passionFruit) [reply src/searchModal.js:265]: Yes, there is a method for that in _tao-core-sdk-fe/src/util/shortcut/registry.js:147_
But as I see, it is not used in many places in tao-core-ui-fe 
Screenshot

Should I create tech debt issue in Jira for refactoring this in other places?

Comment (404passionFruit) [reply src/searchModal.js]: Please check my refactoring

**Review by habemuscode — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ]  New code is respecting code style rules
- [ ]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**